lua/luarocks-tag-release.lua

```diff
@@ -92,7 +93,17 @@ local function luarocks_tag_release(package_name, package_version, specrev, args
       .. ' --api-key $LUAROCKS_API_KEY'
       .. luarocks_extra_flags_and_args
     print('UPLOAD: ' .. cmd)
-    local stdout, _ = OS.execute(cmd, error, args.is_debug)
+    local stdout, _ = OS.execute(cmd, function(message)
+      if message:find("already exists on the server") and not args.fail_on_duplicate then
+        print(
+          string.format(
+            '%s already exists with version %s on the remote. Doing nothing (`fail_on_duplicate` is false).'
+          )
+        )
+      else
+        error(message)
+      end
+    end, args.is_debug)
```

Comment:
Do let me know if it's more in your style to break this out into a separate function :)

Reply:
I'm fine with this as it is.